src/mixxx.cpp

```diff
@@ -598,6 +596,11 @@ void MixxxMainWindow::initializeWindow() {
     Pal.setColor(QPalette::Background, MenuBarBackground);
     m_pMenuBar->setPalette(Pal);
 
+    // restore window size/position
+    QSettings settings("mixxxdj", "mixxx");
```

Comment:
> I was getting a compile error... toBase64() returns a QByteArray and there was no method that takes a QByteArray. I can post more info tomorrow if necessary.

Ah, you may need an explicit typecast to QString then.
http://doc.qt.io/qt-4.8/qstring.html#QString-9